stories/datePicker.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTable from "./components/propTable";
+import { storiesOf } from '@storybook/react';
+import { checkA11y } from 'storybook-addon-a11y';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, boolean, number, select } from "@storybook/addon-knobs/react";
+import { DatePicker } from '../components';
+import { DateTime } from 'luxon';
+import MomentLocaleUtils from 'react-day-picker/moment';
+
+const languages = ['da', 'de', 'fr', 'en', 'es', 'fi', 'it', 'nl', 'pt', 'pl', 'sv'];
+const sizes = ['small', 'medium', 'large'];
+
+const preSelectedDate = DateTime.local().plus({days: 3}).toJSDate();
+
+
+storiesOf('DatePicker', module)
+		.addDecorator((story, context) => withInfo({TableComponent: PropTable})(story)(context))
```

Comment:
This is indented with tabs instead of spaces (and our linter isn't linting the stories, maybe we should lint those as well).